src/style.css

```diff
@@ -0,0 +1,84 @@
+/* No passed !!! */
+
+body {
+  margin: 0;
+  font-family: "Roboto", "sans-serif";
+}
+
+button {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;/* identical to box height */
+  text-align: center;
+  text-transform: uppercase;
+  color: #fff;
+  background: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  width: 100%;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  margin-top: 20px;
+}
+
+button:hover {
+  color: #00acdc;
+  background: #fff;
+  box-sizing: border-box;
+}
+
+.card {
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product {
+  padding: 30px 20px;

```

Comment:
It's unexpected behavior with `padding` on the image. Better to wrap it in separate block and set styles for the wrapper block.